src/pages/App.tsx

```diff
@@ -307,6 +310,9 @@ export default function App() {
                     <Route path={`${APP_PATHS.GRANT_PROGRAMS}`} element={<GrantProgramPage />} />
                     <Route path={`${APP_PATHS.GRANT_PROGRAMS}/:slug`} element={<GrantProgramPage />} />
 
+                    <Route path={`${APP_PATHS.ELASTIC_LEGACY}`} element={<ElasticLegacy />} />
+                    <Route path={`elastic-swap`} element={<ElasticSwap />} />
```

Comment:
why dis not use APP_PATHS

Reply:
just a hidden page to support QA. dont use it anywhere in code